client/src/components/overview/ProductDetails.test 2.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+
+describe('ProductDetails', () => {
+	test.only('the simplest test', () => {
+    const aDiv = render(<div />);
+
+    expect(aDiv).toBeTruthy();
+  });
+});
```

Comment:
Dupe file. Rend it like the abomination it is.
![image](https://user-images.githubusercontent.com/42557448/128584841-d5e7018b-7835-4a5e-aada-f352969675e4.png)